sky/packages/sky/lib/widgets/framework.dart

```diff
@@ -110,6 +110,10 @@ abstract class GlobalKey extends Key {
     assert(removed);
   }
 
+  static Widget getWidget(GlobalKey key) {
+    return _registry[key];
```

Comment:
We should probably assert that key != null here.  It seems like an error to call this function with a null key.